### PR TITLE
Move release to its own releasenotes template (#6823)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description
This is to facilitate making the template more general and moving the release specific logic into its own template.

Trying to break down https://github.com/mozilla/bedrock/pull/6880 into a smaller PR

## Issue / Bugzilla link
#6823

## Testing
Here are examples of each release note product and channel pre and post Quantum. Probably don't need to check them all for this PR though:
- Release
  - http://localhost:8000/en-US/firefox/65.0/releasenotes/ 
  - http://localhost:8000/en-US/firefox/35.0/releasenotes/
- ESR
  - http://localhost:8000/en-US/firefox/52.7.1/releasenotes/
- Beta
  - http://localhost:8000/en-US/firefox/66.0beta/releasenotes/
  - http://localhost:8000/en-US/firefox/55.0beta/releasenotes/
- DevEd Fx? 2014 - Fx54 April 2017
  - http://localhost:8000/en-US/firefox/54.0a2/auroranotes/
- Aurora (Fx5 2011)
  - http://localhost:8000/en-US/firefox/32.0a2/auroranotes/
- Nightly
  - http://localhost:8000/en-US/firefox/67.0a1/releasenotes/
  - http://localhost:8000/en-US/firefox/55.0a1/releasenotes/ (bug-id)
- Android
  - http://localhost:8000/en-US/firefox/android/60.0.2/releasenotes/
  - http://localhost:8000/en-US/firefox/android/55.0.2/releasenotes/
- Android Beta
  - http://localhost:8000/en-US/firefox/android/66.0beta/releasenotes/
  - http://localhost:8000/en-US/firefox/android/55.0beta/releasenotes/
- Android Aurora
  - http://localhost:8000/en-US/firefox/android/54.0a2/auroranotes/
- iOS
  - http://localhost:8000/en-US/firefox/ios/10.0/releasenotes/
  - http://localhost:8000/en-US/firefox/ios/3.0/releasenotes/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/releases/release-notes.html [hunk 1]
```diff
@@ -2,392 +2,4 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
-{% from "macros.html" import google_play_button with context %}
```
Comment (pmac) [inline bedrock/firefox/templates/firefox/releases/release-notes.html]: What is this template for now that the logic has moved to `notes.html`? I don't see it being used anywhere else.

**Comment by stephaniehobson** [comment]
It's one small step of a larger plan to make the if/else logic more readable. I thought it was more reviewable than the 850 lines of code in #6880 ;)

**Comment by stephaniehobson** [comment]
Updated a few more spots. 

I'm having trouble telling what needs to be updated in base.py and would appreciate your guidance.

Part of the problem is it's hard to tell the difference between stuff that is interacting with the release notes database vs the template.

Am I introducing a confusing naming problem? Can you think of a better way to name it? `release-notes.html` and `release-release-notes.html`? `release-notes.html` and `releases-notes.html`?

File: bedrock/firefox/templates/firefox/releases/release-notes.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/releases/release-notes.html]: It's supposed to be used for release Firefox, Firefox for Android, and Firefox iOS.

Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/releases/release-notes.html]: bedrock/bedrock/releasenotes/views.py line 31 handles this I think?

**Comment by stephaniehobson** [comment]
Oh 💩 broke the tests.

File: bedrock/firefox/templates/firefox/releases/release-notes.html (hunk 1 above)
Comment (pmac) [reply bedrock/firefox/templates/firefox/releases/release-notes.html]: OOOOOOOOOhhhh got it. Silly me. I knew I was missing something 👍

**Comment by stephaniehobson** [comment]
Tests passing.

**Comment by pmac** [comment]
@stephaniehobson I'm not sure if you want this to merge now or later, so I've just approved it and you can merge when you'd like.

**Pull request merged**